R/generateCI.R

```diff
@@ -230,7 +230,7 @@ generateCI <- function(stimuli, responses, baseimage, rdata, participants=NA,
     if(zmapmethod == 't.test') {
 
       # Compute one CI in one single step based on all data
-      if (all(is.na(participants))) {
+      if (all(is.na(participants))|!(length(unique(as.numeric(factor(participants)== 1))))) {
```

Comment:
Likewise, can't we just use length(unique(participants)) for this instead of first converting to factor and then numeric?

Also, should `!` be there?